tests/template/test_radial_fourier_template.py spurious failures

I sometimes get a failure like this when running the tests locally:

```
>       assert np.allclose(results["dominant_0"], expected["dominant_0"].raw_data)
E       assert False
E        +  where False = <function allclose at 0x7f7633a078b0>(array([[10,  2,  2,  2, 22],\n       [ 2,  4,  8,  8, 22],\n       [ 2,  4,  8,  8,  2],\n       [ 2,  4,  0,  8,  0],\n       [ 0,  8,  2,  4,  0]]), array([[10,  2,  2,  2,  2],\n       [ 2,  4,  8,  8, 22],\n       [ 2,  4,  8,  8,  2],\n       [ 2,  4,  0,  8,  0],\n       [ 0,  8,  2,  4,  0]]))
E        +    where <function allclose at 0x7f7633a078b0> = np.allclose
E        +    and   array([[10,  2,  2,  2,  2],\n       [ 2,  4,  8,  8, 22],\n       [ 2,  4,  8,  8,  2],\n       [ 2,  4,  0,  8,  0],\n       [ 0,  8,  2,  4,  0]]) = <AnalysisResult: dominant_0>.raw_data
```
The test code here looks suspicious:

https://github.com/LiberTEM/LiberTEM/blob/bc74cd74d55702962b566eeb0b3cd78c6550c8c9/tests/template/test_radial_fourier_template.py#L36-L54

I think the analysis-creation should be outside of the loop, as the analysis doesn't depend on the channel. But that won't solve the problem at hand, I think. Can you comment on this, @twentyse7en ?

--- a/src/libertem/analysis/radialfourier.py
+++ b/src/libertem/analysis/radialfourier.py
@@ -67,7 +67,7 @@ class RadialTemplate(GeneratorHelper):
 
     def get_save(self):
         save = []
-        channels = ["dominant_0", "absolute_0_0", "absolute_0_1"]
+        channels = ["absolute_0_0", "absolute_0_1"]
         for channel in channels:
             result = f"radial_result['{channel}'].raw_data"
             save.append(f"np.save('radial_result_{channel}.npy', {result})")

--- a/tests/template/test_radial_fourier_template.py
+++ b/tests/template/test_radial_fourier_template.py
@@ -34,7 +34,6 @@ def test_radial_fourier_default(hdf5_ds_2, tmpdir_factory, lt_ctx):
     ep = ExecutePreprocessor(timeout=600)
     out = ep.preprocess(nb, {"metadata": {"path": datadir}})
     channels = [
-        "dominant_0",
         "absolute_0_0",
         "absolute_0_1"
     ]
@@ -54,6 +53,5 @@ def test_radial_fourier_default(hdf5_ds_2, tmpdir_factory, lt_ctx):
                                 )
     expected = lt_ctx.run(analysis)
 
-    assert np.allclose(results["dominant_0"], expected["dominant_0"].raw_data)
     assert np.allclose(results["absolute_0_0"], expected["absolute_0_0"].raw_data)
     assert np.allclose(results["absolute_0_1"], expected["absolute_0_1"].raw_data)

